### ISSUE TITLE
marker view not working.

### ISSUE THREAD

**Issue by ashwani-daffodil** [Pod-Point-Open-Source/react-native-maps#5]
i am just passing this array ( in ios ) 

```javascript
markers={[
    {
        id: 'marker-100',
        latitude: 28.608597,
        longitude: 76.656160,
        icon: require('./images/marker.png'),
    }
]}
```

**Comment by edcs** [comment]
Hi - there was a recent API change in React which has stopped markers from working properly. We've got a patch for it and it'll be released in about a day.

**Comment by ashwani-daffodil** [comment]
I can not find where setMarker function and also other function are called in file (PPTGoogleMap.m). i am beginner in ios programming. So i don't know much about it. 

any help would be appreciated.

**Comment by edcs** [comment]
This issue is now fixed in `v1.0.1` - there's no explicit `setMarker` function in the ObjC code, the way the React Native bridge works is quite complex so if you'd like to learn more then I suggest reading this article from the docs as the guys who wrote the package will do a far better job of explaining it than I!: 

http://facebook.github.io/react-native/docs/native-components-ios.html#content
